helm/prometheus-rules/templates/alerting-rules/kvm.management-cluster.rules.yml

```diff
@@ -12,6 +12,17 @@ spec:
   groups:
   - name: kvm
     rules:
+    - alert: BlackboxExporterEndpointDown
+      annotations:
+        description: '{{`Blackbox exporter endpoint ({{ $labels.instance }}) is down.`}}'
+        opsrecipe: blackbox-exporter-endpoint-down/
```

Comment:
I'll add this Ops Recipe while I wait for review.

Reply:
Recipe here https://github.com/giantswarm/prometheus-rules/pull/59

Reply:
The link to the recipe is this PR and not the recipe PR :D